lib/iris/__init__.py

```diff
@@ -121,7 +121,7 @@ def callback(cube, field, filename):
 
 
 # Iris revision.
-__version__ = '2.0.0rc1'
+__version__ = '2.1.0dev1'
```

Comment:
`dev1` not `dev0`?

Reply:
https://www.python.org/dev/peps/pep-0440/#id40

Agreed.